trade-finance-manager-ui/server/controllers/utilisation-reports/find-reports-by-year/search-input-validator.ts

```diff
@@ -0,0 +1,60 @@
+import { isNonEmptyString } from '@ukef/dtfs2-common';
+import { REGEX } from '../../../constants';
+
+type Error = {
```

Comment:
This probably shouldn't be called `Error` as the `Error` type definition is already taken by the javascript `Error` class

Reply:
Changed to `ErrorSummaryItem`